packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts

```diff
@@ -249,15 +262,19 @@ export class AmplifyAuthTransform extends AmplifyCategoryTransform {
       dependsOn: [], // to support undefined meta in update,
     };
 
+    if (this._cliInputs.cognitoConfig.hostedUI) {
+      (parameters as $TSAny).oAuthSecretsPathAmplifyAppId = getAppId();
```

Comment:
this value needs to go in the `team-provider-info.json` file to avoid merge conflicts because it varies across environments in different accounts

Reply:
Nice catch!! .thank you